numba/tests/test_lists.py

```diff
@@ -554,6 +559,13 @@ def test_iteration(self):
     def test_reverse(self):
         self.check_unary_with_size(list_reverse)
 
+    def test_reversed(self):
+        self.disable_leak_check()
+        pyfunc = list_reversed
+        cfunc = jit(nopython=True)(pyfunc)
+        self.assertEquals(pyfunc(4), cfunc(4))
```

Comment:
pytest raised a DeprecationWarning for `assertEquals`. The current function is `assertEqual`